mne/io/fieldtrip.py

```diff
@@ -0,0 +1,334 @@
+# -*- coding: UTF-8 -*-
+# Copyright (c) 2018, Thomas Hartmann & Dirk Gütlin
+# Authors: Thomas Hartmann <thomas.hartmann@th-ht.de>
+#          Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+# License: BSD (3-clause)
+
+import mne
+import numpy as np
+from ..externals import pymatreader
```

Comment:
then you'll need to nest this import wherever you want to use `pymatreader` to avoid always importing it (which again will create a hard h5py dependency that we don't want)